content/docs/get-started/aws/modify-program.md

```diff
@@ -12,176 +12,153 @@ menu:
 aliases: ["/docs/quickstart/aws/modify-program/"]
 ---
 
-Now that we have an instance of our Pulumi program deployed, let's enable encryption on our S3 bucket.
+Now that your S3 Bucket is provisioned, let's add an object to your bucket. First let's create a new directory called `site`.
 
-Replace the entire contents of {{< langfile >}} with the following:
+```bash
+$ mkdir site
+```
+
+Next let's create an `index.html` file you will upload to your bucket.
+
+{{< chooser os "macos,linux,windows" / >}}
+
+{{% choosable os macos %}}
+
+```bash
+$ touch site/index.html
+```
+
+{{% /choosable %}}
+
+{{% choosable os linux %}}
+
+```bash
+$ touch site/index.html
+```
+
+{{% /choosable %}}
+
+{{% choosable os windows %}}
+
+```bash
+$ type nul > site/index.html
+```
+
+{{% /choosable %}}
+
+Once you've created your `index.html` file, add some content to it:
+
+```html
+<html>
+    <body>
+        <h1>Hello, Pulumi!</h1>
+    </body>
+</html>
+```
+
+Now that you have your new `index.html` with some content, let's modify your program to add the contents of your `index.html` file to your S3 Bucket. To accomplish this we will take advantage of your chosen programming language's native libraries to read the contents of your `index.html` and assign the contents as an input to your new `BucketObject`.
```

Comment:
```suggestion
Now that you have your new `index.html` with some content, let's modify your program to add the contents of your `index.html` file to your S3 bucket. To accomplish this, we will take advantage of your chosen programming language's native libraries to read the content of the file and assign it as an input to a new  `BucketObject`.
```

Since this is the first mention of a BucketObject, it's slightly disorienting to refer to it as "your new BucketObject" because it hasn't been introduced -- it's like, "What new BucketObject? Did I miss something?" Whereas if you say, "a new BucketObject", then it's like, "Ah, okay -- I guess that's a thing."